### PR TITLE
feat: add flutter engineer spec

### LINKED ISSUE #6: Flutter engineer spec

### PR DESCRIPTION
**Pull request by alif16syafri**
resolves #6

### CODE REVIEW

File: Mobile-Engineer-Flutter.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+- Possess strong debugging skills
+- Familiar with Git workflow and its principles
+- Familiar with Agile and Scrum
+
+## Preferred Qualifications
+
+- Understanding in reactive programming (RxDart) with BLoC pattern
+- Have performed either Unit test or Integration Test
+- Deployed an flutter application to the Play Store and App Store
+- Knowledge of CI/CD such as Bitrise
```
Comment (nazieb) [inline Mobile-Engineer-Flutter.md]: no need to mention specific product here

File: Mobile-Engineer-Flutter.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+## Qualifications
+
+- Kurio's values (Get sh*t done, Be Bold, Stay Curious, Excellence, Collaboration)
+- Able to write scalable mobile applications using Dart
+- Have knowledge in Flutter's life cycle
+- Have knowledge in Software Architecture such as MVP or MVVM. 
+- Have knowledge of Object-Oriented Programming (OOP)
+- Have at least one basic understading of Android or iOS development
+- Have at least one basic understading of Android Material Design or Cupertino Design principles
```
Comment (nazieb) [inline Mobile-Engineer-Flutter.md]: typo

File: Mobile-Engineer-Flutter.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
+<img src="logo_kurio.png" alt="Logo" style="width: 100px;"/>
+
+# Mobile Engineer - Flutter
+<!-- please add some nice and cool intro here -->
+
+## What you will do
+
+- Build and maintain production-level flutter apps
```
Comment (nazieb) [inline Mobile-Engineer-Flutter.md]:
```suggestion
- Build and maintain production-level Flutter apps
```

File: Mobile-Engineer-Flutter.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+- Have knowledge of SQL
+- Understand the very basic conecept of RESTful API
+- Possess strong debugging skills
+- Familiar with Git workflow and its principles
+- Familiar with Agile and Scrum
+
+## Preferred Qualifications
+
+- Understanding in reactive programming (RxDart) with BLoC pattern
+- Have performed either Unit test or Integration Test
```
Comment (nazieb) [inline Mobile-Engineer-Flutter.md]:
```suggestion
- Have performed either unit test or integration test
```

File: Mobile-Engineer-Flutter.md [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+- Have at least one basic understading of Android Material Design or Cupertino Design principles
+- Have knowledge of SQL
+- Understand the very basic conecept of RESTful API
+- Possess strong debugging skills
+- Familiar with Git workflow and its principles
+- Familiar with Agile and Scrum
+
+## Preferred Qualifications
+
+- Understanding in reactive programming (RxDart) with BLoC pattern
```
Comment (nazieb) [inline Mobile-Engineer-Flutter.md]:
```suggestion
- Understands reactive programming (RxDart) with BLoC pattern
```

File: Mobile-Engineer-Flutter.md [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+- Have performed either Unit test or Integration Test
+- Deployed an flutter application to the Play Store and App Store
```
Comment (nazieb) [inline Mobile-Engineer-Flutter.md]:
```suggestion
- Deployed a Flutter application to the Play Store and App Store
```

**Review by nazieb — CHANGES_REQUESTED** [review]

File: Mobile-Engineer-Flutter.md [hunk 7]
```diff
@@ -0,0 +1,33 @@
+<img src="logo_kurio.png" alt="Logo" style="width: 100px;"/>
+
+# Mobile Engineer - Flutter
+<!-- please add some nice and cool intro here -->
```
Comment (nazieb) [inline Mobile-Engineer-Flutter.md]:
```suggestion
At Kurio, we always love to build our app enthusiastically with next-generation technologies. We strive to enable users to access informations faster without abandoning the beautiful clean UI, clean design, and responsiveness. We're looking for engineers who bring fresh ideas to collaborate with our passionate mobile developers.

## The Role

You will build and maintain our cross-platform app written using Flutter. We expect a good coding standard that results in a good working and maintainable app. As the technology keep evolving, there are always things to learn. We expect each engineers to be able to follow what happens in mobile development ecosystem and share them among team members.
```

**Review by nazieb — APPROVED** [review]

**Review by frendyguo-zz — APPROVED** [review]
LGTM

**Pull request merged**